src/main/webapp/styles/style.css

```diff
@@ -4,8 +4,14 @@
 #map {
   height: 600px;
   margin: auto;
+  width: 75%;
  }
 
+.tutorial_placeholders {
```

Comment:
Nit: It is recommended to use a hyphen instead of underscore

Reply:
+1